app/controllers/employees_controller.rb

```diff
@@ -0,0 +1,60 @@
+class EmployeesController < ApplicationController
+  before_action :fetch_employee, only: %i[show update destroy edit]
+
+  def index
+    @employees = Employee.all.order(id: :asc)
+  end
+
+  def show; end
+
+  def new
+    @employee = Employee.new
+    @address = Address.new
+  end
+
+  def create
+    @employee = Employee.new(employee_params)
+    @employee.hobbies_data = params['employee']['hobbies']
+    if @employee.save
+      redirect_to employees_path
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def edit
+    @address = Address.find_by(employee_id: @employee.id)
+  end
+
+  def update
+    @employee.hobbies_data = params['employee']['hobbies']
+    if @employee.update(employee_params)
+      redirect_to employee_path(@employee)
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @employee.destroy
+
+    redirect_to employees_path
+  end
+
+  def search
+    @employees = Employee.where('employee_name LIKE ?', '%' + params[:search_employee] + '%')
```

Comment:
@employees = Employee.where('employee_name LIKE ?', "%#{params[:search_employee]}%")
    render :index
    You can do it like this to avoid security issues